joomla/joomla-cms
# [4.0] Update Bootstrap to 4.4.1

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/27287.

### Summary of Changes

Updates Bootstrap from 4.3.1 to 4.4.1 and Popper.js from 1.15.0 to 1.16.0.

### Testing Instructions

Run `npm install`.
Browse around frontend and backend.
Check that styling still OK.
Check that Bootstrap JS components (modal, dropdowns, popovers) still work.

### Expected result

Works.

### Documentation Changes Required

No.
